ci-operator/step-registry/dpf/hypervisor/deploy-cluster/dpf-hypervisor-deploy-cluster-commands.sh

```
@@ -136,30 +136,32 @@ else
-echo "File ${REMOTE_MAIN_WORK_DIR}/env/env.user_${CLUSTER_NAME} was found on hypervisor"
+echo "Copying user-env from Vault cluster profile to hypervisor..."
+if ! scp ${SSH_OPTS} "${USER_ENV_FILE}" root@${REMOTE_HOST}:${REMOTE_WORK_DIR}/openshift-dpf/user.env; then
+  echo "ERROR: Failed to copy user-env to hypervisor"
+  exit 1
+fi
 
-echo "Copy the env.user file in ${REMOTE_MAIN_WORK_DIR}/env to ${REMOTE_WORK_DIR}/openshift-dpf, source the file, then generate .env file"
-if ssh ${SSH_OPTS} root@${REMOTE_HOST} "cp ${REMOTE_MAIN_WORK_DIR}/env/env.user_${CLUSTER_NAME} ${REMOTE_WORK_DIR}/openshift-dpf; \
-  cd ${REMOTE_WORK_DIR}/openshift-dpf; \
+echo "Generating .env file from Vault-provided user.env..."
+if ssh ${SSH_OPTS} root@${REMOTE_HOST} "cd ${REMOTE_WORK_DIR}/openshift-dpf; \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)